runtime/src/loader_utils.rs

```diff
@@ -206,6 +212,48 @@ pub fn load_upgradeable_program(
     });
 }
 
+pub fn load_upgradeable_program_wrapper(
+    bank_client: &BankClient,
+    mint_keypair: &Keypair,
+    authority_keypair: &Keypair,
+    name: &str,
+) -> Pubkey {
+    let buffer_keypair = Keypair::new();
+    let program_keypair = Keypair::new();
+    load_upgradeable_program(
+        bank_client,
+        mint_keypair,
+        &buffer_keypair,
+        &program_keypair,
+        authority_keypair,
+        name,
+    );
+    program_keypair.pubkey()
+}
+
+pub fn load_upgradeable_program_and_advance_slot(
```

Comment:
Instead of removing the dev-context-only-utils from other places, I think it
needs to be added here? And possibly in other places too

Reply:
See comments above.

Reply:
Yes. Added.